lib_eio/multipart_form_eio.mli

```diff
@@ -0,0 +1,63 @@
+open Multipart_form
+
+(** {3 Streaming API.} *)
+
+val stream :
+  sw:Eio.Switch.t ->
+  ?bounds:int ->
+  identify:(Header.t -> 'id) ->
+  string Eio.Stream.t ->
+  Content_type.t ->
+  ('id t, [> `Msg of string ]) result Eio.Promise.t
+  * ('id * Header.t * string Eio.Stream.t) Eio.Stream.t
+(** [stream ~identify src content_type] returns:
+    - a promise [th] about the parser
+    - a stream [stream] of parts
+
+    They can be manipulated with [Lwt.both], and, by this way,
```

Comment:
You probably should update the documentation as well and describe how to use this function with `eio`.

Reply:
The documentation has been updated now. Thanks for your feedback!